client/i18n/locales/english/intro.json

```diff
@@ -1696,14 +1696,302 @@
     "title": "Front End Development",
     "intro": ["placeholder"],
     "blocks": {
-      "0": {
-        "title": "0",
-        "intro": []
-      },
-      "1": {
-        "title": "1",
-        "intro": []
+      "0": { "title": "0", "intro": [] },
```

Comment:
I cannot make a proper suggestion here. The `intro.json` should remain exactly as before adding the new block (order of blocks included) except for the placeholder `196` which should be replaced by 
```
"build-a-real-time-counter": {
        "title": "Build a Real Time Counter",
        "intro": ["", ""]
      },
```

Reply:
I have copied the original `intro.json` from the main repo, replaced the placeholder 196.
I hope this is how the file should look like.